sdk/servicebus/service-bus/src/util/utils.ts

```diff
@@ -215,11 +203,11 @@ export function getInteger(value: any, nameOfProperty: string): number {
  * or undefined if not passed in.
  */
 export function getIntegerOrUndefined(value: any): number | undefined {
-  if (value == undefined) {
+  if (!isDefined(value)) {
     return undefined;
   }
   const result = parseInt(value.toString());
-  return result == NaN ? undefined : result;
+  return isNaN(result) ? undefined : result;
```

Comment:
Did the linter catch this or did you? This is a good catch either way, our runtime logic was actually wrong!

Reply:
The linter caught it